packages/core-components/src/utils/dom-utils.ts

```diff
@@ -0,0 +1,105 @@
+import { isFocusable, isHidden } from '@a11y/focus-trap/focusable';
+
+/*
+ * The MIT License (MIT)
```

Comment:
IIRC we need to call this out in a `THIRD_PARTY_DEPENDENCIES` file - as we're embedding another person/company's source code.

Reply:
Is there any reason why the license/comment isn't at the very top of the file?